src/pages/CookieClicker.vue

```diff
@@ -31,19 +84,53 @@ const cps = computed(() => {
 
 </script>
 <template>
-    <div class="columns">
-        <div class="column is-3 has-background-primary has-text-centered">
-            <p class="is-size-1">Cookies: {{  +parseFloat(cookies).toFixed( 1 ) }}</p>
-            <p class="is-size-5">CPS: {{ +parseFloat(cps).toFixed( 1 ) }}</p>
-            <figure @click="cookieClick" class="image is-1by1 m-5">
+
+    <div class="columns" id="CookieClicker">
+        <div class="column is-3 has-background--bulma-primary-00 has-text-centered ">
+            <div v-for="(cookie, index) in fallingCookies" :key="index" class="cookie-rain" :style="{
+                left: cookie.left,
+                animationDuration: cookie.animationDuration,
+                animationDelay: cookie.animationDelay
+            }">
                 <img src="https://pngimg.com/uploads/cookie/cookie_PNG13656.png" />
-            </figure>
+            </div>
+            <div class="container  p-6">
+                <p class="is-size-1">{{ +parseFloat(cookies).toFixed(1) }} cookies</p>
+                <p class="is-size-5">cookies per second: {{ +parseFloat(cps).toFixed(1) }}</p>
+                <figure @click="cookieClick" :class="{ press: press }" class="image is-1by1 m-5 cookie-container ">
+                    <img src="https://pngimg.com/uploads/cookie/cookie_PNG13656.png" draggable="false" />
+                </figure>
+            </div>
         </div>
-        <div class="column has-background-info">
-
+ <div class="column has-primary-00 ">
+        <div v-for="building in buildings.slice(1)" :key="building.name" class="column" :class="{'box': building.count > 0}">
+            <div class="is-flex is-align-items-center overflow-hidden">
+                <figure v-for="(index) in Math.min(building.count, 22)" :key="index" class="image is-48x48">
+                    <img :src="building.img" />
+                </figure>
+            </div>
         </div>
-        <div class="column is-2 has-background-warning">
-            <button v-for="building in buildings" :disabled="cookies < building.price" @click="buyBuilding(building)" class="button is-primary is-large is-fullwidth">{{ building.name }} {{ building.price }} {{ building.count }}</button>
+    </div>
+
+
+        <div class="column is-2 has-background--bulma-primary-00">
+            <button v-for="building in buildings" :disabled="cookies < building.price" @click="buyBuilding(building)"
+                class="button is-text is-large is-fullwidth under">
+                <figure v-if="building.img" class="image is-64x64">
+                    <img :src="building.img" alt="" />
+                </figure> {{ building.name }} {{ building.price }} {{ building.count }}
+            </button>
         </div>
     </div>
-</template>
\ No newline at end of file
+</template>
+
+<style scoped>
+#CookieClicker {
+    height: 94vh;
```

Comment:
better solve it by adding 100% to body and other stuff on the way bcs 94vh only works with certain window size